src/code_e0b30_len_b80.c

```diff
@@ -4,7 +4,10 @@ INCLUDE_ASM("code_e0b30_len_b80", get_default_variation_for_song);
 
 INCLUDE_ASM("code_e0b30_len_b80", func_8014A498);
 
-INCLUDE_ASM("code_e0b30_len_b80", func_8014A52C);
+void func_8014A52C(void) {
+    D_80159AE2 = (s16)0x08;
```

Comment:
These casts shouldn't be needed since these variables are already s16s.

Reply:
Sounds good, made the change under [this commit](https://github.com/ethteck/papermario/pull/45/commits/dedd132adcdb5683dbff926209d3e87d36e87354).